### DISCUSSION TITLE
Use my own domain for GitHub pages and still use it to point my VPS

### DISCUSSION THREAD

**Discussion by ehguille** [community/community#23552] [Other Feature Feedback, Questions, & Ideas]
<p>Hi there,</p>
<p>I’m planning using my own domain for GitHub pages, but I still would like to use it to store my stuff.</p>
<p>I mean: I would like <a href="http://domain.com" rel="noopener nofollow ugc">domain.com</a> point to my pages on GitHub, but still be able to access to my files subfolders in my webserver (<a href="http://domain.com/photos" rel="noopener nofollow ugc">domain.com/photos</a>, and so on).</p>
<p>I guess it’s no possible,  may I be wrong?</p>

**Comment by jfullstackdev** [comment]
<aside class="quote no-group" data-username="ehguille" data-post="1" data-topic="226037">
<div class="title">
<div class="quote-controls"></div>
 ehguille:</div>
<blockquote>
<p>I would like <a href="http://domain.com">domain.com</a> point to my pages on GitHub, but still be able to access to my files subfolders in my webserver (<a href="http://domain.com/photos">domain.com/photos</a>, and so on)</p>
</blockquote>
</aside>
<p>you will be having your own domain name, is that what you mean?</p>
<p>after that, you will be creating a GitHub Page but instead of the default domain, you will be using your own domain,</p>
<p>now, you want to store files at your webserver as you mentioned, what is that webserver? is that Amazon? or any other else? because you can simply store your files on GitHub too, provided they are not violating any rules of GitHub,</p>
<p>if you want to store your files outside GitHub, on your own webserver, then you can always embed it on GitHub, GitHub being the host of your web page,</p>

**Comment by ehguille** [comment]
<p>Thank you for your help.</p>
<p>Is more like I want to have my pages on <a href="http://domain.com" rel="noopener nofollow ugc">domain.com</a>, but still be able to serve some apps on <a href="http://domain.com/folder1" rel="noopener nofollow ugc">domain.com/folder1</a>, folder2, etc.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>That’s not possible. The GitHub servers would have to act as a reverse proxy for your site, which isn’t supported.</p>
